util/uvmdvgen/gen_env.py

```diff
@@ -55,17 +55,21 @@ def gen_env(name, is_cip, has_ral, has_interrupts, has_alerts, env_agents,
         src_suffix = tup[3]
 
         # Skip Makefile
-        if src == 'Makefile' and not add_makefile: continue
+        if src == 'Makefile' and not add_makefile:
+            continue
```

Comment:
Please remove this as well - seems unnecessary now.

Reply:
You can remove `add_makefile` arg in the function and remove the corresponding switch in `uvmdvgen.py` as well.